pkg/splunk/enterprise/util.go

```diff
@@ -211,6 +211,85 @@ func GetSmartstoreRemoteVolumeSecrets(volume enterprisev1.VolumeSpec, client spl
 	return accessKey, secretKey, namespaceScopedSecret.ResourceVersion, nil
 }
 
+// ApplyAppListingConfigMap creates the configMap  with two entries:
+// (1) app-list-local.yaml
+// (2) app-list-cluster.yaml
+func ApplyAppListingConfigMap(client splcommon.ControllerClient, cr splcommon.MetaObject,
+	appConf *enterprisev1.AppFrameworkSpec, appsSrcDeployStatus map[string]enterprisev1.AppSrcDeployInfo) (*corev1.ConfigMap, bool, error) {
+
+	var err error
+	var crKind string
+	var configMapDataChanged bool
+	crKind = cr.GetObjectKind().GroupVersionKind().Kind
+
+	scopedLog := log.WithName("ApplyAppListingConfigMap").WithValues("kind", crKind, "name", cr.GetName(), "namespace", cr.GetNamespace())
+
+	mapAppListing := make(map[string]string)
+
+	// ToDo: sgontla/Jeff: Adapt to any Ansible changes(if required) for differentiating local vs. cluster scope (?)
```

Comment:
Yes this will need to be addressed.  Currently if we put both the cluster and the local in both, for CM/Deployer all apps regardless of scope will have be pushed to the cluster and disabled locally. 

Sidenote: do we block use of `scope: cluster` for non clustered Types (anything other than SHC and CM)?